src/keycloak.ts

```diff
@@ -3,6 +3,6 @@ import Keycloak from "keycloak-js";
 
 export const keycloakInstance = new Keycloak({
   realm: process.env.REACT_APP_KEYCLOAK_REALM,
-  url: process.env.REACT_APP_KEYCLOAK_WORKFLOWS_API,
+  url: process.env.REACT_APP_KEYCLOAK_URL,
```

Comment:
why?

Reply:
I made changes to the previous one `REACT_APP_KEYCLOAK_WORKFLOWS_API`.

Reply:
why are you changing this? this doesn't need to be changed?

Reply:
I mistakenly changed that, sorry.